src/lang/en-US.json

```diff
@@ -845,17 +845,30 @@
   "time_past": "{0} ago",
   "time_just_now": "just now",
   "time_s": "a second",
+  "time_abbr_s": "{0}s",
   "time_ss": "{0} seconds",
+  "time_abbr_ss": "{0}s",
   "time_m": "a minute",
+  "time_abbr_m": "{0}m",
   "time_mm": "{0} minutes",
+  "time_abbr_mm": "{0}m",
   "time_h": "an hour",
+  "time_abbr_h": "{0}h",
   "time_hh": "{0} hours",
+  "time_abbr_hh": "{0}h",
   "time_d": "a day",
+  "time_abbr_d": "{0} day",
```

Comment:
think we can just use d here

Reply:
and I don't think we need both singular and plural abbreviations, just one should be good

Reply:
I was just considering multiple languages in case they need to express singular/mutliple days differently.

Reply:
ok.  I think day should still be d for both in english